src/test/integration/utils/mocha.js

```diff
@@ -136,6 +137,13 @@ async function initDebugger(url, ...sources) {
   return dbg;
 }
 
+function countSources(dbg) {
+  const sources = dbg.selectors.getSources(dbg.getState());
+
+  // the web test runner has one extra source because it injects the debuggee script
+  return sources.size - 1;
```

Comment:
This might be just me :) but the navigation tests seem to be failing  because this, when there are no sources
![screen shot 2017-02-23 at 5 29 12 pm](https://cloud.githubusercontent.com/assets/792924/23271212/9db1469e-f9ee-11e6-93e4-5221bac5fadf.png)